index.md

```diff
@@ -1,11 +1,4 @@
-<p align="center">
-	<a href="https://cambiatus.com">
-    <img src="https://cambiatus.github.io/assets/logo-cambiatus.png" alt="logo" width="300"> 
-	</a>
-</p>
-<br/>
-
-## Welcome to Cambiatus Wiki :wave:
```

Comment:
I really liked the logo on here

Reply:
I will change it back! Reason for removing was because of the wiki page layout it looked crowded as it has a logo on the left and this one on the top.